pkgs/misc/emulators/dgen-sdl/default.nix

```
@@ -0,0 +1,55 @@
+{ stdenv
+, fetchurl
+, libarchive
+, doxygen
+, SDL
+}:
+
+let
+  pname = "dgen-sdl";
+  version = "1.33";
+in stdenv.mkDerivation {
+  inherit pname version;
```

```suggestion
in stdenv.mkDerivation  rec {
  pname = "dgen-sdl";
  version = "1.33";
```

Please stop doing this.

---

Why?